include/digitalPins.h

```diff
@@ -0,0 +1,39 @@
+#ifndef __DIGITAL_H__
+#define __DIGITAL_H__
+
+#include "ina281.h"
+#include "PID.h"
+
+//Define macros for the digital pins
+
+//Outputs
+#define MCU_DIR                PA_6
+#define MCU_ECO                PB_1
+#define MCU_MC_ON              PA_2
+
+//Inputs
+#define MCU_SPEED_SIG          PA_8
+#define PRK_BRK_TELEM          PB_4
+
+struct Digital_Data {
+    bool direction : 1;         // output
+    bool mc_speed_sig : 1;      // input
+    bool eco_mode : 1;          // output
+    bool mc_on : 1;             // output
```

Comment:
Let's rename this to `mcu_mc_on` so it matches the macro and the name in the spreadsheet.